src/hooks/commentsHooks/useGetComments.ts

```diff
@@ -0,0 +1,22 @@
+import { useQuery } from "react-query"
+import type { UseQueryResult } from "react-query"
+
+import { COMMENTS_KEY } from "constants/queryKeys"
+
+import * as CommentsService from "services/CommentsService"
+
+import { CommentData, CommentProps } from "types/comments"
+
+export const useGetComments = ({
+  siteName,
+  requestId,
+}: CommentProps): UseQueryResult<CommentData[]> => {
+  return useQuery<CommentData[]>(
+    [COMMENTS_KEY, siteName],
+    () => CommentsService.getComments({ siteName, requestId }),
+    {
+      retry: false,
+      enabled: false, // Manually triggered
```

Comment:
As discussed offline - the main purpose was to prevent refetching on focus, which would clear the unread status of comments - we've opted to use `refetchOnWindowFocus:false` instead https://github.com/isomerpages/isomercms-frontend/pull/1102/commits/01641ddb3d79e2a9fbe6bb786c1e8f068ce27956